2025-02-08 11:38:49,858 - INFO - Running on host/machine: kamiche.polytechnique.fr
2025-02-08 11:38:49,859 - INFO - Command used to run the script: prompt_engineering.py --repo_folder ../original_repo --csv_folder data/all_csv/ --dataset tokenized_data/test_examples.json --learning_type zero_shot --format_type markdown,naturalized,json,html --models deepseek-r1:32b --parallel_models --batch_prompts --max_workers 10
2025-02-08 11:38:49,893 - INFO - Loaded data from ../original_repo/tokenized_data/test_examples.json.
2025-02-08 11:38:49,893 - INFO - Loaded data from ../original_repo/tokenized_data/test_examples.json.
2025-02-08 11:38:49,893 - INFO - Loaded data from ../original_repo/tokenized_data/test_examples.json.
2025-02-08 11:38:49,893 - INFO - Loaded data from ../original_repo/tokenized_data/test_examples.json.
2025-02-08 11:38:49,948 - INFO - Ollama started successfully (or was already running).
2025-02-08 11:38:49,948 - INFO - Ollama started successfully (or was already running).
2025-02-08 11:38:49,948 - INFO - Ollama started successfully (or was already running).
2025-02-08 11:38:49,949 - INFO - Ollama started successfully (or was already running).
2025-02-08 11:38:49,950 - INFO - Processing 1436 tables in parallel (skipping 259 already checkpointed) out of 1436 remaining.
2025-02-08 11:38:49,950 - INFO - Processing 1435 tables in parallel (skipping 260 already checkpointed) out of 1435 remaining.
2025-02-08 11:38:49,950 - INFO - Processing 1437 tables in parallel (skipping 258 already checkpointed) out of 1437 remaining.
2025-02-08 11:38:49,951 - INFO - Processing 1443 tables in parallel (skipping 252 already checkpointed) out of 1443 remaining.
2025-02-08 11:39:03,552 - INFO - HTTP Request: POST http://127.0.0.1:11434/api/generate "HTTP/1.1 200 OK"
2025-02-08 11:39:07,523 - INFO - HTTP Request: POST http://127.0.0.1:11434/api/generate "HTTP/1.1 200 OK"
2025-02-08 11:39:09,282 - INFO - HTTP Request: POST http://127.0.0.1:11434/api/generate "HTTP/1.1 200 OK"
2025-02-08 11:39:09,284 - INFO - HTTP Request: POST http://127.0.0.1:11434/api/generate "HTTP/1.1 200 OK"
2025-02-08 11:39:17,365 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,388 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,406 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,409 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,426 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,441 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,447 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,449 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,548 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,578 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,586 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,595 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,598 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,599 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,609 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,609 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,610 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,611 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,614 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,614 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,616 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,619 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,626 - ERROR - Error invoking LLM: [Errno 111] Connection refused
2025-02-08 11:39:17,629 - ERROR - Error invoking LLM: [Errno 111] Connection refused
